.github/workflows/repos-mirror.yml

```diff
@@ -0,0 +1,24 @@
+name: MindSpore Gitee repos mirror periodic job
+
+on:
+  pull_request:
+    # Runs at every pull requests submitted in master branch 
+    branches: [ master ]
```

Comment:
does this mean  watching every pull request creat event or merge event?

Reply:
Yes, so I recomened that we should remove L4-L7 when the workflow is stable, this is just for test in this pr, otherwise, the job will not be triggered.